build.gradle

```diff
@@ -190,9 +191,16 @@ subprojects {
             signature 'net.sf.androidscents.signature:android-api-level-14:4.0_r4@signature'
             signature 'org.codehaus.mojo.signature:java18:1.0@signature'
         }
+
+        // Add dependency to the BOM artifact
```

Comment:
Please add a more detailed description of why this is needed, e.g. "Add dependency on kotlinx-serialization-bom inside other kotlinx-serialization modules themselves, so they...."